src/test/java/ladder/domain/LadderTest.java

```diff
@@ -35,11 +38,18 @@ void getWidthTest() {
     @Test
     @DisplayName("특정 좌표에 스틱이 존재하는지 알 수 있다")
     void isExistTest() {
-        LineGenerator lineGenerator = size -> new Line(sticks);
-        Ladder ladder = Ladder.makeLadder(new Height(3), 2, lineGenerator);
+        LinePatternGenerator lineGenerator = new LinePatternGenerator(new AlwaysTrueSupplier());
+        Ladder ladder = Ladder.makeLadder(new Height(3), 3, lineGenerator);
 
-        boolean actual = ladder.isExist(2, 1);
+        boolean actual = ladder.isExist(1, 0);
 
-        assertThat(actual).isFalse();
+        assertThat(actual).isTrue();
+    }
+
+    class AlwaysTrueSupplier implements BooleanSupplier {
```

Comment:
```suggestion
    static class AlwaysTrueSupplier implements BooleanSupplier {
```
static nested class 와 non-static nested class 의 차이는 무엇일까요?

Reply:
고민해 본적 없는 주제였는데 집어 주셔서 감사해요..

이팩티브 자바에서 static 내부 클래스를 선언하라고 강조 하더라구요.

그 이유를 찾아보니 non-static 내부 클래스라면 안쪽 클래스에서 외부 클래스의 인스턴스 변수를 참조할 수 있더군요

이런 숨은 참조 떄문에 GC에 대상에 들어가지 않고 메모리 누수가 일어날 수 있다고 학습했습니다.

Reply:
이펙티브 자바까지 읽으시다니! 👍